docs/pages/dropdown.js

```diff
@@ -97,11 +97,11 @@ card(
       },
 
       {
-        name: 'dangerouslyRemoveLayer',
+        name: 'isWithinFixed',
         type: 'boolean',
         defaultValue: false,
         description:
-          'Removes the Layer component around Popover. Should only be used in cases where Layer breaks the Dropdown positionings such as when the anchor element is within a sticky component.',
+          'Enables correct behavior when the Dropdown is within a fixed container. To achieve this it removes the Layer component around Popover and enables position relative to anchor. Should only be used in cases where Layer breaks the Dropdown positionings such as when the anchor element is within a sticky component.',
```

Comment:
nit: "enables positioning relative to its anchor element."

Reply:
nit: we typically don't use articles when referring to components in the abstract, so this should read "Enables correct behavior when Dropdown is used within a fixed container."

Reply:
Fixed